scripts/configure-venue-access.ts

```diff
@@ -8,16 +8,30 @@ import "firebase/firestore";
 
 import { VenueAccessMode } from "../src/types/VenueAcccess";
 
-import { initFirebaseAdminApp, makeScriptUsage } from "./lib/helpers";
+import {
+  initFirebaseAdminApp,
+  makeScriptUsage,
+  FieldValue,
+} from "./lib/helpers";
 
 const usage = makeScriptUsage({
   description: "Configures the venue access with the selected method and value",
-  usageParams:
-    "PROJECT_ID VENUE_ID [password|emaillist|codelist] [password | emails file path | codes file path] [CREDENTIAL_PATH]",
-  exampleParams:
-    "co-reality-map password abc123 [theMatchingAccountServiceKey.json] / co-reality-map emails emails-one-per-line.txt [theMatchingAccountServiceKey.json] / co-reality-map codes ticket-codes-one-per-line.txt [theMatchingAccountServiceKey.json]",
+  usageParams: [
+    "PROJECT_ID VENUE_ID METHOD ACCESS_DETAIL [CREDENTIAL_PATH]",
+    "",
+    `PROJECT_ID VENUE_ID ${VenueAccessMode.Password} THE_PASSWORD_TO_SET [CREDENTIAL_PATH]`,
+    `PROJECT_ID VENUE_ID ${VenueAccessMode.Emails} LIST_OF_EMAILS_PATH [CREDENTIAL_PATH]`,
+    `PROJECT_ID VENUE_ID ${VenueAccessMode.Codes} LIST_OF_ONE_TIME_USE_CODES_PATH [CREDENTIAL_PATH]`,
+  ],
+  exampleParams: [
+    `co-reality-map venue123Id ${VenueAccessMode.Password} secretPassword123 [theMatchingAccountServiceKey.json]`,
+    `co-reality-map venue123Id ${VenueAccessMode.Emails} emails-one-per-line.txt [theMatchingAccountServiceKey.json]`,
+    `co-reality-map venue123Id ${VenueAccessMode.Codes} one-time-use-ticket-codes-one-per-line.txt [theMatchingAccountServiceKey.json]`,
+  ],
 });
 
+// TODO: refactor this to work with multiple venueIds at once, in a similar way to how scripts/count-users-entered-venue.ts works
```

Comment:
[TODO] refactor this to work with multiple `venueIds` at once, in a similar way to how `scripts/count-users-entered-venue.ts` works

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1455/commits/f138ce8941565ea6b8db40d82d1c040af625b478